src/test/java/org/example/BaseTests.java

```diff
@@ -42,25 +46,42 @@ public class BaseTests {
 	protected RemoteWebDriver driver;
 	protected HomePage homePage;
 	protected WebDriverWait wait;
+	private static final String userName = "juan_ocampo";
+	private static final String password = "Jira_password01";
```

Comment:
look at:
- https://www.baeldung.com/maven-arguments#passing-an-argument-to-maven
- https://stackoverflow.com/questions/7513319/passing-command-line-arguments-from-maven-as-properties-in-pom-xml#:~:text=So%20these%20properties%20can%20be%20accessed%20from%20the%20Java%20code%3A

Reply:
Looking forward I want to tip you with check if those args are mandatory and make verification if there is the any value setted if not throug the exception and and let the executor know that those args is mandatory.

Reply:
Variables in pom.xml file and called in BaseTests file